R/runner.R

```diff
@@ -467,17 +467,24 @@ orderly_runner_ <- R6::R6Class(
     kill = function(key) {
       task_id <- self$con$HGET(self$keys$key_task_id, key)
       if (is.null(task_id)) {
-        porcelain::porcelain_stop(
-          sprintf("Failed to kill '%s' task doesn't exist", key))
+        return(list(
+          killed = FALSE,
+          message = sprintf("Failed to kill '%s' task doesn't exist", key)
```

Comment:
```suggestion
          message = sprintf("Failed to kill '%s'\n   task doesn't exist", key)
```

for consistency with below?